### ISSUE TITLE
[Question] Are Animania pigs still a subtype of vanilla pigs?

### ISSUE THREAD

**Issue by WesCook** [capnkirok/animania#579]
Sorry for cluttering your issues list, but there's no source code for me to check this myself.

I'm using the [Better Questing 2](https://minecraft.curseforge.com/projects/better-questing) mod to detect entities around the player for a quest.  As Animania introduces many variants, I can't have a quest for each one.  Instead I've been using the vanilla version of the mob.

![screenshot from 2018-02-21 19 26 48](https://user-images.githubusercontent.com/792101/36518887-353f318a-173d-11e8-8b5a-f25731baccb3.png)

As Animania mobs inherited from the vanilla mob, this still worked correctly with each variant of the animal, and the entity was detected correctly.  However, this behavior appears to have broken in one of the updates specifically for pigs.  Other animals are still detected correctly by their vanilla version.

Has Animania changed how inheritance works in regards to mob subtypes?  If so, would there be any way to make this functionality work again?

Relevant bug on my modpack's tracker: https://github.com/WesCook/Homestead/issues/85

Thank you.

**Comment by capnkirok** [comment]
Hey @WesCook !

Yeah.... the pig is not an extension of vanilla. I was having problems controlling pig riding so I don't extend it. It actually never was extended from vanilla. How much of an issue is this? I can try and find another way to handle pig riding, but it may take a bit of work.

**Comment by WesCook** [comment]
Thanks for the information.

>It actually never was extended from vanilla. 

Must have been sloppy testing on my part then!  I thought I verified them all as working but it's definitely possible I missed one.

>How much of an issue is this?

It's not pack-breaking by any means.  I may have to remove the quest, or just add a technical note that detection doesn't work for this animal.  But it's not something I expect you to make a large technical overhaul just to fix.

Ultimately my approach was a bit of a hack anyway, but I just wanted to better understand the problem before I attempted to tackle it on my end.  If a simple fix was possible on your end too I thought I'd ask about it.

If you do decide to reimplement pig riding and thus inherit from vanilla, a quick update to this issue would be appreciated.  :)  Otherwise, thanks for the great mod and enjoy your day.

Thanks!
